modules/rubiconBidAdapter.js

```diff
@@ -949,6 +868,81 @@ function addVideoParameters(data, bidRequest) {
   data.imp[0].video.h = size[1]
 }
 
+function applyFPD(bidRequest, mediaType, data) {
+  const bidFpd = {
+    user: {...bidRequest.params.visitor},
+    context: {...bidRequest.params.inventory}
+  };
+
+  if (bidRequest.params.keywords) bidFpd.context.keywords = (utils.isArray(bidRequest.params.keywords)) ? bidRequest.params.keywords.join(',') : bidRequest.params.keywords;
```

Comment:
Likewise, adunit/imp-level 'keywords' should go into opentrb imp.ext.rubicon.keywords